docs/clients/throttle_source_manager.rst

```diff
@@ -0,0 +1,123 @@
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+
+    % Make a ThrottleSourceManager object with obj_id 0
+    SourceManager = ThrottleSourceManager(’com’,com);
+
+    % Use the ThrottleSourceManager object
+    dronecan_throttle_prio = SourceManager.get(dronecan_priority);
+
+Python
+~~~~~~
+
+To use the Throttle Source Manager Client in Python, import ``iqmotion`` and create a module that has the Throttle Source Manager Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the Throttle Source Manager Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0) |module_name_comment|
+    
+    dronecan_prio = |variable_name|.get("throttle_source_manager", "dronecan_priority") 
+    print(f"DroneCAN Priority: {dronecan_prio}")
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 104 | Throttle Source Manager
+    .. table:: Throttle Source Manager
+        :widths: 8 18 15 10 5 50
+        :class: tight-table
+
+        +--------+------------------------+----------------+-----------+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | Sub ID | Short Name             | Access         | Data Type | Unit                            | Note                                                                                                                                                                              |
+        +========+========================+================+===========+=================================+===================================================================================================================================================================================+
+        | 0      | throttle_timeout       | get, set, save | float     | Seconds                         | Defines the amount of time, in seconds, that the Throttle Source Manager will wait for a new message from the highest received priority before checking the next lowest priority. |
+        +--------+------------------------+----------------+-----------+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 1      | dronecan_priority      | get, set, save | uint8     |                                 | The relative priority of DroneCAN throttles against hobby and IQUART throttles. Must be a value [0, 3] where 0 means DroneCAN is ignored, and 3 is the highest priority.          |

```

Comment:
Should the Unit for the priority values be Enum?

Reply:
Nope, the priorities are just numbers, they don't map to any special treatment